weave/tests/test_basket.py

```diff
@@ -737,3 +736,107 @@ def test_read_only_get_data():
         del read_only_pantry
         del read_only_fs
         del my_basket
+
+
+def test_create_basket_in_place(test_pantry):
+    """Test create basekt in place works without a pantry.
+    """
+    # although we are not testing a pantry upload it
+    # still makes sense to use the test_pantry for clean-up
```

Comment:
Write out as full sentence with capitalization and punctuation. I think we also avoid using "we" in the comments/docstrings.